docs/reference/api/docker_remote_api_v1.14.md

```diff
@@ -126,7 +126,11 @@ Create a container
              "Tty":false,
              "OpenStdin":false,
              "StdinOnce":false,
-             "Env":null,
+             "Env": [
+                     "FOO=bar",
+                     "FOO=baz",
```

Comment:
wasn't sure if I should use that, but the issue is that it is allowed (last one takes effect). Do you think I should remove the dups?